package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "build": "cross-env NODE_ENV=production rollup -c rollup.config.js",
     "minify": "cross-env NODE_ENV=production rollup -c rollup-minify.config.js",
-    "start": "npm run build && http-server ."
+    "start": "npm run build && http-server . -c-1"
```

Comment:
I recommend add `-c-1` option to disable cache. With this change the change of the files on the server will have an effect even without rebooting the server